docs/essentials/exceptions/exceptions-create-user-defined.md

```diff
@@ -1,6 +1,6 @@
 # How to create user-defined exceptions
 
-If you want users to be able to programmatically distinguish between some error conditions, you can create your own user-defined exceptions. The .NET Framework provides a hierarchy of exception classes ultimately derived from the base class [Exception](https://msdn.microsoft.com/library/system.exception). Each of these classes defines a specific exception, so in many cases you only have to catch the exception. You can also create your own exception classes by deriving from the **Exception** class.
+If you want users to be able to programmatically distinguish between some error conditions, you can create your own user-defined exceptions. .NET Core provides a hierarchy of exception classes ultimately derived from the base class [Exception](http://dotnet.github.io/api/System.Exception.html). Each of these classes defines a specific exception, so in many cases you only have to catch the exception. You can also create your own exception classes by deriving from the **Exception** class.
 
 When creating your own exceptions, it is good coding practice to end the class name of the user-defined exception with the word "Exception." It is also good practice to implement the three recommended common constructors, as shown in the following example.
 
```

Comment:
We've been doing it like this:

```
> **Note**
>
> content content content content content
```

Which renders to this:

> **Note**
>
> content content content content content

[Example](http://dotnet.github.io/docs/languages/csharp/syntax.html)